include/ggml/ggml.h

```diff
@@ -1483,6 +1505,12 @@ extern "C" {
             int                  d1,
             bool                 is_2D);
 
+    GGML_API struct ggml_tensor * ggml_depthwise_conv(
+        struct ggml_context * ctx,
+        struct ggml_tensor  * a,
+        struct ggml_tensor  * b,
+        int                   p0);
+
     GGML_API struct ggml_tensor * ggml_conv_1d(
```

Comment:
There is an implementation available on `master`:

https://github.com/ggerganov/ggml/blob/6b14d738d9100c50c199a3b1aaa960f633904476/include/ggml/ggml.h#L1499-L1510